### DISCUSSION TITLE
Album Name in Thumbnail & Link Previews while sharing album links to Non-Ente Users

### DISCUSSION THREAD

**Discussion by haritvsmurali** [ente-io/ente#4090] [Enhancements]
While Sharing Album links to Non Ente Users the Thumbnail generated has collage of pics in the album and the count of the total photos. Would be good if it shows the NAME of the album in the thumbnail and in the link previews.

**Comment by chiraag-nataraj** [comment]
There must be some way to generate metatags that are added client-side (so the server doesn't have to know anything about the album). I'm just not familiar enough to know which metatags need to be set...
